src/Calendar.css

```diff
@@ -0,0 +1,55 @@
+@media only screen and (max-width: 600px) {
```

Comment:
Based on the name of this CSS file, I would assume we don't need it (if it indeed was for the Calendar component).

Reply:
This has been removed